grid/chat/members/maps/discord.suma

```diff
@@ -0,0 +1,187 @@
+profile = "chat/members@1.0"
+provider = "discord"
+
+"""
+GetMembers map
+"""
+map GetMembers {
+  return map error if (input.server === undefined) {
+    title = "Input parameter `server` is required in case of discord provider"
+  }
+
+  http GET "/guilds/{input.server}/members" {
+    request "application/json" {
+      query {
+        after = input.page
+        limit = input.limit
+      }
+
+      headers {
+        "Authorization" = `Bot ${parameters.accessToken}`
+      }
+    }
+
+    response 200 "application/json" {
+      members = call MapMembers(members = body)
+      rateLimit = call MapRateLimitDetails(headers = headers)
+
+      result.members = members,
+      result.nextPage = members[members.length - 1].id
+
+      set if (rateLimit) {
+        result.rateLimit = rateLimit
+      }
+
+      map result result
+    }
+
+    response {
+      error = call MapDiscordError(
+        statusCode = statusCode,
+        error = body,
+        headers = headers,
+      )
+      
+      map error error
+    }
+  }
+}
+
+"""
+Operation MapMembers
+Returns array mapping members in received body
+"""
+operation MapMembers {
+  members = []
+
+  members = args.members.map(member => {
+    const resultMember = {
+      id: member.user.id,
+      joinedAt: Date.parse(member.joined_at),
```

Comment:
Watch out, this will return a ms timestamp!

Reply:
Is that a problem?

Reply:
Sure it isn't if you think that year `36681` is the same as `2004` (see `1095379198.75` vs `1095379198750`).

The documentation explicitly states UNIX timestamps giving me example in seconds, so...